src/assets/plus.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 
 const Plus = (props) => {
-  let dimensions = (props.small ? 24 : 32)
+  let dimensions = (props.large ? 30 : 24)
```

Comment:
![image](https://user-images.githubusercontent.com/24930524/87859700-228e6c00-c905-11ea-8e72-37a713e3e9d2.png)
I think it looks a bit off on mobile... maybe the icons should be smaller on smaller screens too?

Reply:
it also might be easier instead of passing in the large prop to just pass in classname as a prop (see the other svg assets for examples on how to do that) so that it's easier to adjust styling for different cases